index.html

```diff
@@ -1035,17 +1035,13 @@
 
    <section>
      <h2>Braille</h2>
-       <p>There is an important difference between just sending the raw text to
-         a braille display which will translate the text to the default braille
-         grade and language the braille display is using and having a pre-
-         formatted braille rendition using a specific braille table.</p>
+       <p>There is an important difference between just sending the raw text to a braille display which will translate the text to the default braille grade and language the braille display is using and having a pre-formatted braille rendition using a specific braille table.</p>
      <ul class="use-cases">
-        <li>As a publisher wanting to fulfill my legal requirements in creating
-          an accessible Math textbook, I wish to include my math equation
-          examples in Braille as well as MathML since not all reading systems
-          fully support MathML.  Nemeth Braille is primarily used so I have
-          added separate preformatted Nemeth Braille examples to my publication
-          in addition to the MathML.</li>
+        <li>As a publisher wanting to fulfill my legal requirements in creating an accessible Math textbook, I wish to include my math equation examples in Braille as well as MathML since not all reading systems fully support MathML.  Nemeth Braille is primarily used so I have added separate preformatted Nemeth Braille examples to my publication in addition to the MathML.</li>
```

Comment:
Just wondering... I think that in the newer use cases we tried to avoid using personal voice (or whatever the proper English term is for that), ie, avoid using 'I wish'. Would it be necessary/possible to change that to impersonal one?

WDYT?